api/metrics/urls.py

```diff
@@ -5,6 +5,7 @@
 app_name = 'osf'
 
 urlpatterns = [
+    url(r'^raw/(?P<params>[a-z0-9._/]*)$', views.RawMetricsView.as_view(), name=views.RawMetricsView.view_name),
```

Comment:
I don't think this is good name for the regex variable, you want is the url path, not the query params.